src/middleware/validateSchema.js

```diff
@@ -29,7 +28,6 @@ module.exports = (shouldUseJoiError = false) => {
 		if (includes(supportedMethods, method) && has(schemas, route)) {
```

Comment:
this will be taken out. I mean the if statement surrounding the code because now the schema will be passed as a parameter